### DISCUSSION TITLE
Website not shutting down

### DISCUSSION THREAD

**Discussion by maybeSwarup** [vercel/community#2802] [Help]
### Summary

Hi,

On vercel, a website was deployed i want to take it down, the website is this: https://mhv-frontend-2-0.vercel.app
I stopped it from hosting from settings but that didn't work then I just deleted the project but that also didn't work, can someone from vercel please help me out.

### Steps to Reproduce

deploy a website from vercel dashboard

**Comment by smaeda-ks** [comment]
Hi @swarvar, 

> I just deleted the project but that also didn't work

The project is still live. Can you double-check?

**Reply by maybeSwarup** [reply]
Yes I did, for me the project doesn't exist anymore, I literally just deleted it, but still the link is there @smaeda-ks
